### DISCUSSION TITLE
How to use a theme that I just added to the repo?

### DISCUSSION THREAD

**Discussion by tiagaoalb** [rajasegar/alacritty-themes#66] [General]
I uninstalled the package and installed again, but the palenight theme is not there.
What should I do?

**Comment by JuanVqz** [comment]
Good question I was about to let you know that, your theme won't be available until the next release, we and almost all packages (at least the ones I have been working) work with releases.

I'm not sure if we should release a new 4.1.x patch when we introduce a new theme, @rajasegar what do you think?

BTW, for that, you should stay stick to the commitlint standard.

**Reply by JuanVqz** [reply]
@tiagaoalb let's wait for the owner answer, please

**Comment by JuanVqz** [comment]
@tiagaoalb well, we just released [alacritty-themes v4.1.4](https://github.com/rajasegar/alacritty-themes/releases/tag/v4.1.4)

Now you can install it globally, I'd suggest using it from the source code that way you will have the latest version always, though

**Comment by JuanVqz** [comment]
The answer to this is: After we merged the theme you proposed just download the latest version and it should be there.  :+1:
